### ISSUE TITLE
Text suggestion for plan wizard page - funder 

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2885]
**What version of the DMPRoadmap code are you running? (e.g. v2.2.0)**
v.2.2.0

When entering funder name and the funder is not listed it currently says 
<img width="613" alt="Screenshot 2021-04-23 at 16 45 32" src="https://user-images.githubusercontent.com/47178106/115896593-7d845f00-a453-11eb-9b1d-2eb0cab0935e.png">

It has been suggested that it would be more helpful to have “There is not yet a template for this funder” or something similar as a text - do you have any thoughts @mariapraetzellis @pherterich

**Comment by mariapraetzellis** [comment]
Yes, that text makes sense to me. How about updating the message to read: 
"There are no templates for this funder" . Okay with you @pherterich?
